mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.cpp

```diff
@@ -323,6 +322,9 @@ void CLeaderboardsTimes::SetPlaceColors(SectionedListPanel* panel, TimeType_t ty
     if (itemCount == 0)
         return;
 
+    // We need to force the panel to resort its items now
+    m_pLocalLeaderboards->InvalidateLayout(true, false);
```

Comment:
`false` isn't needed for the second param, they both default to false.

```suggestion
    m_pLocalLeaderboards->InvalidateLayout(true);
```